aws/resource_aws_lb_listener_test.go

```diff
@@ -521,7 +628,7 @@ resource "aws_alb_listener" "front_end" {
 
 resource "aws_alb" "alb_test" {
   name            = "%s"
-  internal        = true
+  internal        = false
```

Comment:
I think this change broke the existing acceptance test:

```
--- FAIL: TestAccAWSLBListener_BackwardsCompatibility (21.17s)
    testing.go:568: Step 0 error: errors during apply:
        
        Error: Error creating application Load Balancer: InvalidSubnet: VPC vpc-063a62e8cef2e74f3 has no internet gateway
```

I will revert it on merge as it looks unrelated/unexpected. 👍

Reply:
Sorry about that - that's a weird regression I added to that test!